src/payment/data/selectors.js

```diff
@@ -62,6 +62,7 @@ export const paymentSelector = createSelector(
         basket.loaded && !basket.redirect && (!basket.products || basket.products.length === 0),
       isRedirect:
         (basket.loaded && !!basket.redirect) || (!basket.loaded && isCouponRedeemRedirect),
+      captureKeyId: basket.captureContext ? basket.captureContext.keyId : '',
```

Comment:
It's a bit of a nit, but I'd favor `null` here instead of `''` - we don't really use the latter for falsy values very much.